.gitignore

```diff
@@ -8,3 +8,5 @@ frontend_keys.js
 .DS_Store
 ngrok
 bundle.js.gz
+frontend_keys.js
+secretsBackend.js
```

Comment:
@AnnaC91  the file name for the secrets file is secrets_backend.js please update here and make sure your files match

Reply:
I've been keeping all of the old file names on git ignore in case swtiching between older and newer branches causes accidental leaks, etc.